mpbb-checkout

```diff
@@ -190,6 +198,7 @@ checkout() {
 # Do not edit !!!
 sources_conf ${option_work_dir}/sources.conf
 host_blacklist *.distfiles.macports.org *.packages.macports.org
+archive_site_local ${option_archive_sites}
```

Comment:
I thought I had tested `archive_site_local` on my system, but I was doing it wrong. Turns out it [doesn't work at all](
https://trac.macports.org/ticket/57717); it's been broken for years. That bug in base will need to be fixed, and the one that [doesn't allow multiple URLs](https://trac.macports.org/ticket/57718) in `archive_site_local`. And mpbb-checkout will need to be changed to append `:tbz2` to each URL.